### PR TITLE
Redesign cards on the homepage

### PR DESCRIPTION
**Pull request by mjoyce91**
Redesigns the cards on the homepage based on recent mockups. There are still some small content changes that have yet to be made. We also use a static Time of Estimated Departure.

Another more noticeable change I made here is the gutter width. USWDS default max-width for the grid is 1040px, so I bumped that up to 1200px to more accurately reflect the the wireframes without going too far into full responsiveness.

I also updated the user dropdown to use the user's first name instead of their username.

**#1270 should be merged first or the dashboard will break!!**

### CODE REVIEW

File: src/Components/CondensedCardData/CondensedCardData.jsx [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
+      title="Skill Code"
+      content={position.skill || NO_SKILL}
+      hasFixedTitleWidth
+    />
+    <CondensedCardDataPoint
+      title="Language"
+      content={<LanguageList languages={position.languages} />}
+      hasFixedTitleWidth
+    />
+    <StaticDevContent>
```
Comment (burgwyn) [inline src/Components/CondensedCardData/CondensedCardData.jsx]: 👍

File: src/Components/CondensedCardData/CondensedCardDataPoint/CondensedCardDataPoint.test.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  it('applies the correct class when hasFixedTitleWidth is true', () => {
+    const wrapper = shallow(
+      <CondensedCardDataPoint
+        {...props}
+        hasFixedTitleWidth
+      />,
+    );
+    expect(wrapper.find('.condensed-card-data-title').exists()).toBe(true);
+    expect(wrapper.find('.title-fixed-width').exists()).toBe(true);
```
Comment (burgwyn) [inline src/Components/CondensedCardData/CondensedCardDataPoint/CondensedCardDataPoint.test.jsx]: 👍

File: src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx [hunk 3]
```diff
@@ -1,59 +1,38 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import FontAwesome from 'react-fontawesome';
-import ResultsNewFlag from '../ResultsNewFlag';
-import Favorite from '../Favorite/Favorite';
-import { NO_LAST_UPDATED_DATE } from '../../Constants/SystemMessages';
-import { POSITION_DETAILS, FAVORITE_POSITIONS_ARRAY } from '../../Constants/PropTypes';
-import { formatDate } from '../../utilities';
+import { POSITION_DETAILS } from '../../Constants/PropTypes';
```
Comment (burgwyn) [inline src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx]: Let's remove the comments

File: src/Constants/DefaultProps.js [hunk 4]
```diff
@@ -7,8 +7,8 @@ export const DEFAULT_HOME_PAGE_POSITIONS = {
 
 export const DEFAULT_USER_PROFILE = {
   user: {
-    username: '...', // show '...' when loading
-    first_name: '',
+    username: '...',
```
Comment (burgwyn) [inline src/Constants/DefaultProps.js]: Should we move this to `''`?

Comment (mjoyce91) [reply src/Constants/DefaultProps.js]: I've been using `'...'` as an easy way to display that it's loading

Comment (burgwyn) [reply src/Constants/DefaultProps.js]: That's fine

**Pull request merged**